tests/FSharp.Compiler.ComponentTests/Conformance/InferenceProcedures/RecursiveSafetyAnalysis/RecursiveSafetyAnalysis.fs

```diff
@@ -49,7 +49,7 @@ module RecursiveSafetyAnalysis =
             (Error 1114, Line 8, Col 15, Line 8, Col 25, "The value 'E_RecursiveInline.test' was marked inline but was not bound in the optimization environment")
             (Error 1113, Line 7, Col 16, Line 7, Col 20, "The value 'test' was marked inline but its implementation makes use of an internal or private function which is not sufficiently accessible")
             (Warning 1116, Line 8, Col 15, Line 8, Col 25, "A value marked as 'inline' has an unexpected value")
-            (Error 1118, Line 8, Col 15, Line 8, Col 25, "Failed to inline the value 'test' marked 'inline', perhaps because a recursive value was marked 'inline'")
+            (Warning 1118, Line 8, Col 15, Line 8, Col 25, "Failed to inline the value 'test' marked 'inline', perhaps because a recursive value was marked 'inline'")
```

Comment:
@t-Gro, no it's not breaking.  If the developers have warn as error, it will still error, otherwise it will compile.  So no compiling code will break.  The change is that whereas before, non-inlinable would fail to compile with an error, now it will compile with a warning, this will not break code, because, now the inlining won't happen.  I made it a warning so that if a developer required it to be inline, they would be notified that inlining might won't happen.

The change is required because for **realsig-**  inlining a private value will work, because it's promoted to internal, whereas for **realsig+** inlining may not work because private values are only visible for classes and modules nested in the source code hierarchy.  I didn't want to fork the code so that code would compile realsig- and not realsig+, so I relaxed the error to a warning.